[
{"message":"file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/cli/index.js:237\n        throw new Error(y, { cause: u });\n              ^\n","level":"FATAL","type":"PLAYGROUND","test":"ast-sqlite-boot","plugin":"ezinearticles-wordpress-plugin","details":"file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/cli/index.js:237\n        throw new Error(y, { cause: u });\n              ^\n","log":"/tmp/tmp.Mw87C3FfSs"}
,
{"message":"Unknown query type: DESC.","level":"FATAL","type":"SQL","test":"ast-sqlite-boot","plugin":"ezinearticles-wordpress-plugin","details":"[26-May-2025 06:43:00 UTC] WordPress database error <div style=\"clear:both\">&nbsp;</div>\n<div class=\"queries\" style=\"clear:both;margin-bottom:2px;border:red dotted thin;\">\n<p>MySQL query:</p>\n<p>DESC `wp_external_url_as_post_featured_image_thumbnail__errors_log`</p>\n<p>Queries made or created this session were:</p>\n<ol>\n<li>Executing: BEGIN | (no parameters)</li>\n<li>Executing: ROLLBACK | (no parameters)</li>\n</ol>\n</div>\n<div style=\"clear:both;margin-bottom:2px;border:red dotted thin;\" class=\"error_message\" style=\"border-bottom:dotted blue thin;\">\nError occurred at line 4117 in Function <code>handle_error</code>. Error message was: Unknown query type: DESC.\n</div>\n<p>Backtrace:</p>\n<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()\n#1 /wordpress/wp-includes/class-wpdb.php(3107): WP_SQLite_DB->query('DESC `wp_extern...')\n#2 /wordpress/wp-content/plugins/external-url-as-post-featured-image-thumbnail/library_wp.php(1103): wpdb->get_col('DESC `wp_extern...', 0)\n#3 /wordpress/wp-content/plugins/external-url-as-post-featured-image-thumbnail/library_wp.php(1058): Puvox\\library_wp->db_table_columns('wp_external_url...')\n#4 /wordpress/wp-content/plugins/external-url-as-post-featured-image-thumbnail/library_wp.php(1370): Puvox\\library_wp->add_column_my('wp_external_url...', 'function_name', 'longtext', 'gmdate')\n#5 /wordpress/wp-content/plugins/external-url-as-post-featured-image-thumbnail/library_wp.php(1712): Puvox\\library_wp->create_log_table()\n#6 /wordpress/wp-content/plugins/external-url-as-post-featured-image-thumbnail/library_wp.php(1684): Puvox\\wp_plugin->plugin_inits()\n#7 /wordpress/wp-content/plugins/external-url-as-post-featured-image-thumbnail/index.php(530): Puvox\\wp_plugin->__construct()\n#8 /wordpress/wp-settings.php(545): include_once('/wordpress/wp-c...')\n#9 /wordpress/wp-config.php(102): require_once('/wordpress/wp-s...')\n#10 /wordpress/wp-load.php(50): require_once('/wordpress/wp-c...')\n#11 /internal/eval.php(3): require_once('/wordpress/wp-l...')\n#12 {main}</pre>\n for query DESC `wp_external_url_as_post_featured_image_thumbnail__errors_log` made by require_once('wp-load.php'), require_once('wp-config.php'), require_once('wp-settings.php'), include_once('/plugins/external-url-as-post-featured-image-thumbnail/index.php'), Puvox\\wp_plugin->__construct, Puvox\\wp_plugin->plugin_inits, Puvox\\library_wp->create_log_table, Puvox\\library_wp->add_column_my, Puvox\\library_wp->db_table_columns, WP_SQLite_DB->query, WP_SQLite_DB->print_error","log":"/logs/plugins/e/ezinearticles-wordpress-plugin/error.json"}
,
{"message":"SQLSTATE[HY000]: General error: 1 duplicate column name: function_name.","level":"FATAL","type":"SQL","test":"ast-sqlite-boot","plugin":"ezinearticles-wordpress-plugin","details":"[26-May-2025 06:43:00 UTC] WordPress database error <div style=\"clear:both\">&nbsp;</div>\n<div class=\"queries\" style=\"clear:both;margin-bottom:2px;border:red dotted thin;\">\n<p>MySQL query:</p>\n<p>ALTER TABLE `wp_external_url_as_post_featured_image_thumbnail__errors_log` ADD `function_name` longtext NOT NULL AFTER gmdate</p>\n<p>Queries made or created this session were:</p>\n<ol>\n<li>Executing: BEGIN | (no parameters)</li>\n<li>Executing: INSERT INTO _mysql_data_types_cache (`table`, `column_or_index`, `mysql_type`)\n\t\t\t\tVALUES (:table, :column, :datatype)\n\t\t\t\tON CONFLICT(`table`, `column_or_index`) DO UPDATE SET `mysql_type` = :datatype\n\t\t\t | parameters: wp_external_url_as_post_featured_image_thumbnail__errors_log, `FUNCTION_NAME`, longtext</li>\n<li>Executing: ALTER TABLE wp_external_url_as_post_featured_image_thumbnail__errors_log ADD `function_name` text NOT NULL   | (no parameters)</li>\n<li>Executing: ROLLBACK | (no parameters)</li>\n</ol>\n</div>\n<div style=\"clear:both;margin-bottom:2px;border:red dotted thin;\" class=\"error_message\" style=\"border-bottom:dotted blue thin;\">\nError occurred at line 4117 in Function <code>handle_error</code>. Error message was: SQLSTATE[HY000]: General error: 1 duplicate column name: function_name.\n</div>\n<p>Backtrace:</p>\n<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()\n#1 /wordpress/wp-content/plugins/external-url-as-post-featured-image-thumbnail/library_wp.php(1063): WP_SQLite_DB->query('ALTER TABLE `wp...')\n#2 /wordpress/wp-content/plugins/external-url-as-post-featured-image-thumbnail/library_wp.php(1370): Puvox\\library_wp->add_column_my('wp_external_url...', 'function_name', 'longtext', 'gmdate')\n#3 /wordpress/wp-content/plugins/external-url-as-post-featured-image-thumbnail/library_wp.php(1712): Puvox\\library_wp->create_log_table()\n#4 /wordpress/wp-content/plugins/external-url-as-post-featured-image-thumbnail/library_wp.php(1684): Puvox\\wp_plugin->plugin_inits()\n#5 /wordpress/wp-content/plugins/external-url-as-post-featured-image-thumbnail/index.php(530): Puvox\\wp_plugin->__construct()\n#6 /wordpress/wp-settings.php(545): include_once('/wordpress/wp-c...')\n#7 /wordpress/wp-config.php(102): require_once('/wordpress/wp-s...')\n#8 /wordpress/wp-load.php(50): require_once('/wordpress/wp-c...')\n#9 /internal/eval.php(3): require_once('/wordpress/wp-l...')\n#10 {main}</pre>\n for query ALTER TABLE `wp_external_url_as_post_featured_image_thumbnail__errors_log` ADD `function_name` longtext NOT NULL AFTER gmdate made by require_once('wp-load.php'), require_once('wp-config.php'), require_once('wp-settings.php'), include_once('/plugins/external-url-as-post-featured-image-thumbnail/index.php'), Puvox\\wp_plugin->__construct, Puvox\\wp_plugin->plugin_inits, Puvox\\library_wp->create_log_table, Puvox\\library_wp->add_column_my, WP_SQLite_DB->query, WP_SQLite_DB->print_error","log":"/logs/plugins/e/ezinearticles-wordpress-plugin/error.json"}
,
{"message":"Unknown query type: DESC.","level":"FATAL","type":"SQL","test":"ast-sqlite-boot","plugin":"ezinearticles-wordpress-plugin","details":"[26-May-2025 06:43:08 UTC] WordPress database error <div style=\"clear:both\">&nbsp;</div>\n<div class=\"queries\" style=\"clear:both;margin-bottom:2px;border:red dotted thin;\">\n<p>MySQL query:</p>\n<p>DESC `wp_external_url_as_post_featured_image_thumbnail__errors_log`</p>\n<p>Queries made or created this session were:</p>\n<ol>\n<li>Executing: BEGIN | (no parameters)</li>\n<li>Executing: ROLLBACK | (no parameters)</li>\n</ol>\n</div>\n<div style=\"clear:both;margin-bottom:2px;border:red dotted thin;\" class=\"error_message\" style=\"border-bottom:dotted blue thin;\">\nError occurred at line 4117 in Function <code>handle_error</code>. Error message was: Unknown query type: DESC.\n</div>\n<p>Backtrace:</p>\n<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()\n#1 /wordpress/wp-includes/class-wpdb.php(3107): WP_SQLite_DB->query('DESC `wp_extern...')\n#2 /wordpress/wp-content/plugins/external-url-as-post-featured-image-thumbnail/library_wp.php(1103): wpdb->get_col('DESC `wp_extern...', 0)\n#3 /wordpress/wp-content/plugins/external-url-as-post-featured-image-thumbnail/library_wp.php(1058): Puvox\\library_wp->db_table_columns('wp_external_url...')\n#4 /wordpress/wp-content/plugins/external-url-as-post-featured-image-thumbnail/library_wp.php(1370): Puvox\\library_wp->add_column_my('wp_external_url...', 'function_name', 'longtext', 'gmdate')\n#5 /wordpress/wp-content/plugins/external-url-as-post-featured-image-thumbnail/library_wp.php(1712): Puvox\\library_wp->create_log_table()\n#6 /wordpress/wp-content/plugins/external-url-as-post-featured-image-thumbnail/library_wp.php(1684): Puvox\\wp_plugin->plugin_inits()\n#7 /wordpress/wp-content/plugins/external-url-as-post-featured-image-thumbnail/index.php(530): Puvox\\wp_plugin->__construct()\n#8 /wordpress/wp-settings.php(545): include_once('/wordpress/wp-c...')\n#9 /wordpress/wp-config.php(102): require_once('/wordpress/wp-s...')\n#10 /wordpress/wp-load.php(50): require_once('/wordpress/wp-c...')\n#11 /internal/eval.php(3): require_once('/wordpress/wp-l...')\n#12 {main}</pre>\n for query DESC `wp_external_url_as_post_featured_image_thumbnail__errors_log` made by require_once('wp-load.php'), require_once('wp-config.php'), require_once('wp-settings.php'), include_once('/plugins/external-url-as-post-featured-image-thumbnail/index.php'), Puvox\\wp_plugin->__construct, Puvox\\wp_plugin->plugin_inits, Puvox\\library_wp->create_log_table, Puvox\\library_wp->add_column_my, Puvox\\library_wp->db_table_columns, WP_SQLite_DB->query, WP_SQLite_DB->print_error","log":"/logs/plugins/e/ezinearticles-wordpress-plugin/error.json"}
,
{"message":"SQLSTATE[HY000]: General error: 1 duplicate column name: function_name.","level":"FATAL","type":"SQL","test":"ast-sqlite-boot","plugin":"ezinearticles-wordpress-plugin","details":"[26-May-2025 06:43:08 UTC] WordPress database error <div style=\"clear:both\">&nbsp;</div>\n<div class=\"queries\" style=\"clear:both;margin-bottom:2px;border:red dotted thin;\">\n<p>MySQL query:</p>\n<p>ALTER TABLE `wp_external_url_as_post_featured_image_thumbnail__errors_log` ADD `function_name` longtext NOT NULL AFTER gmdate</p>\n<p>Queries made or created this session were:</p>\n<ol>\n<li>Executing: BEGIN | (no parameters)</li>\n<li>Executing: INSERT INTO _mysql_data_types_cache (`table`, `column_or_index`, `mysql_type`)\n\t\t\t\tVALUES (:table, :column, :datatype)\n\t\t\t\tON CONFLICT(`table`, `column_or_index`) DO UPDATE SET `mysql_type` = :datatype\n\t\t\t | parameters: wp_external_url_as_post_featured_image_thumbnail__errors_log, `FUNCTION_NAME`, longtext</li>\n<li>Executing: ALTER TABLE wp_external_url_as_post_featured_image_thumbnail__errors_log ADD `function_name` text NOT NULL   | (no parameters)</li>\n<li>Executing: ROLLBACK | (no parameters)</li>\n</ol>\n</div>\n<div style=\"clear:both;margin-bottom:2px;border:red dotted thin;\" class=\"error_message\" style=\"border-bottom:dotted blue thin;\">\nError occurred at line 4117 in Function <code>handle_error</code>. Error message was: SQLSTATE[HY000]: General error: 1 duplicate column name: function_name.\n</div>\n<p>Backtrace:</p>\n<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()\n#1 /wordpress/wp-content/plugins/external-url-as-post-featured-image-thumbnail/library_wp.php(1063): WP_SQLite_DB->query('ALTER TABLE `wp...')\n#2 /wordpress/wp-content/plugins/external-url-as-post-featured-image-thumbnail/library_wp.php(1370): Puvox\\library_wp->add_column_my('wp_external_url...', 'function_name', 'longtext', 'gmdate')\n#3 /wordpress/wp-content/plugins/external-url-as-post-featured-image-thumbnail/library_wp.php(1712): Puvox\\library_wp->create_log_table()\n#4 /wordpress/wp-content/plugins/external-url-as-post-featured-image-thumbnail/library_wp.php(1684): Puvox\\wp_plugin->plugin_inits()\n#5 /wordpress/wp-content/plugins/external-url-as-post-featured-image-thumbnail/index.php(530): Puvox\\wp_plugin->__construct()\n#6 /wordpress/wp-settings.php(545): include_once('/wordpress/wp-c...')\n#7 /wordpress/wp-config.php(102): require_once('/wordpress/wp-s...')\n#8 /wordpress/wp-load.php(50): require_once('/wordpress/wp-c...')\n#9 /internal/eval.php(3): require_once('/wordpress/wp-l...')\n#10 {main}</pre>\n for query ALTER TABLE `wp_external_url_as_post_featured_image_thumbnail__errors_log` ADD `function_name` longtext NOT NULL AFTER gmdate made by require_once('wp-load.php'), require_once('wp-config.php'), require_once('wp-settings.php'), include_once('/plugins/external-url-as-post-featured-image-thumbnail/index.php'), Puvox\\wp_plugin->__construct, Puvox\\wp_plugin->plugin_inits, Puvox\\library_wp->create_log_table, Puvox\\library_wp->add_column_my, WP_SQLite_DB->query, WP_SQLite_DB->print_error","log":"/logs/plugins/e/ezinearticles-wordpress-plugin/error.json"}
,
{"message":"define(): Argument #3 ($case_insensitive) is ignored since declaration of case-insensitive constants is no longer supported in /wordpress/wp-content/plugins/ezinearticles-wordpress-plugin/wp_ezinearticles.php on line 21","level":"WARNING","type":"PHP","test":"ast-sqlite-boot","plugin":"ezinearticles-wordpress-plugin","details":"[26-May-2025 06:44:30 UTC] PHP Warning:  define(): Argument #3 ($case_insensitive) is ignored since declaration of case-insensitive constants is no longer supported in /wordpress/wp-content/plugins/ezinearticles-wordpress-plugin/wp_ezinearticles.php on line 21","log":"/logs/plugins/e/ezinearticles-wordpress-plugin/error.json"}
,
{"message":"File class-snoopy.php is <strong>deprecated</strong> since version 3.0.0! Use wp-includes/http.php instead. in /wordpress/wp-includes/functions.php on line 6121","level":"DEPRECATED","type":"PHP","test":"ast-sqlite-boot","plugin":"ezinearticles-wordpress-plugin","details":"[26-May-2025 06:44:30 UTC] PHP Deprecated:  File class-snoopy.php is <strong>deprecated</strong> since version 3.0.0! Use wp-includes/http.php instead. in /wordpress/wp-includes/functions.php on line 6121","log":"/logs/plugins/e/ezinearticles-wordpress-plugin/error.json"}
]
